netutils/interface.py

```diff
@@ -294,7 +308,7 @@ def weight(self) -> int:  # noqa: D107,D102
 class CCInt(CharacterClass):
     """Ints must be sorted canonically because '11' < '5'."""
 
-    def __lt__(self, other) -> bool:  # noqa: D105
+    def __lt__(self, other) -> bool:  # type: ignore  # noqa: D105
```

Comment:
Can be fixed with def __lt__(self, other: "CharacterClass") -> bool: # noqa: D105